include/deal.II/numerics/vector_tools.h

```diff
@@ -739,7 +739,6 @@ namespace VectorTools
                                  const ConstraintMatrix              &constraints,
```

Comment:
You're changing the permissions of this file from 644 to 755. Can you undo this?